pages/signup.html

```diff
@@ -76,9 +94,7 @@
           </div>
 
           <div class="d-flex flex-col px-8 py-4">
-            <label class="text-sm my-2" for="email"
-              >Name</label
-            >
+            <label class="text-sm my-2" for="email">Name</label>
```

Comment:
for="email" whereas the label is being used for name

Reply:
yes I will correct it now